packages/stg-devtools-evm-hardhat-v2/src/asset/sdk.ts

```diff
@@ -45,9 +45,22 @@ export class Asset extends Ownable implements IAsset {
             },
         ])
 
+        // TODO add contract name and function name to print log -- @layerzerolabs/devtools
+        // contract name should be in OmniTransaction.OmniPoint.contractName
+        // function name here is setAddressConfig which should be included in description? or its own optional field in OmniTransaction that prints if passed...
+
+        //         Add contract name to log
+        //              exists in OmniTransaction.OmniPoint.contractName so only need to add support to logger? can use devtools formatOmniPoint to list contract name
+        //         Add function being called to log
+        //              likely as an optional param to OmniTransaction first
+        //              then add support to logger?
+
+        // where/how is omnitransaction logged?
+
         return {
             ...this.createTransaction(data),
             description: `Setting address config for ${formatOmniPoint(omniContractToPoint(this.contract))}:\n${printRecord(config)}`,
+            // TODO functionName: 'setAddressConfig',
```

Comment:
I am investigating the best way to add this additional logging.

1. For contract name, I see that `OmniTransaction.OmniPoint` has a `contractName` property that is already fetched with the `formatOmniPoint` method in `devtools` [here](https://github.com/LayerZero-Labs/devtools/blob/36446d0eba516bed98748ea49009efc7b6dafec4/packages/devtools/src/omnigraph/format.ts#L6-L7). Perhaps I can re-use this function within the printed table to replace `OmniAddress`?
2. For function name, I am thinking to add an optional property to `OmniTransaction` so we can return it alongside description [here](https://github.com/stargate-protocol/stargate-v2/blob/26feafc1659f82e0428373e29220e22a7ba5b1ce/packages/stg-devtools-evm-hardhat-v2/src/asset/sdk.ts#L78). Then it should automatically be included in the printed table I think.

My main blocker right now is that I am unsure of which logic _prints_ the below table. It seems to take `OmniTransaction` and prints it in a pretty way. I need to make the following changes to that table logic:
1. Instead of printing `OmniAddress` it should print the contract name (perhaps using `formatOmniPoint`)
2. Also prints `functionName`, which will be an additional property of `OmniTransaction`

![image](https://github.com/user-attachments/assets/8d65cc74-5400-44e6-9169-03157e19d523)

Reply:
The SDK should already have a `label` property that you can use instead of just the `OmniPoint`. 

Besides that, the `OmniPoint` can already have `label` or `contractName` I believe or smth like that on it (if not we can add it). The idea for the SDKs in general is that they are not tied to `hardhat` so just keep that in mind - they can reference a contract without having its source code.

Reply:
Hmm I think I see what you mean -- I am able to access `this.point` in this file, but the `contractName` property is `undefined` (`address` is defined). 

I am looking into where `this.point` is coming from to see if anything needs to be changed there. Calling `omniContractToPoint(this.contract).contractName` also returns `undefined`. 

In the meantime, I have opened up a PR in devtools repo that updates the properties of `OmniTransaction` so that the above log can display all of its properties if you have time to review 🙏  https://github.com/LayerZero-Labs/devtools/pull/991

I will close this PR out soon because I think all changes can be done in devtools -- I believe @calebbanister would like these logs for _every_ transaction so updating a single description in a single sdk config is likely not the way to go i think

Reply:
Ok so this PR will not be closed actually because we need to pass `functionName`. I am still determining why `point.contractName` is undefined!